tensorflow/compiler/xla/refcounting_hash_map_test.cc

```diff
@@ -77,7 +77,7 @@ TEST(RefcountingHashMapTest, CustomFactory) {
 
 TEST(RefcountingHashMapTest, ForEachEmpty) {
   RefcountingHashMap<int, int> m;
-  int64 count = 0;
+  int64_t count = 0;
```

Comment:
Why this change, `int64` is used everywhere throughout the codebase?

Reply:
compile error without this change

```
ERROR: /root/tensorflow/tensorflow/compiler/xla/BUILD:928:1: C++ compilation of rule '//tensorflow/compiler/xla:refcounting_hash_map_test' failed (Exit 1)
tensorflow/compiler/xla/refcounting_hash_map_test.cc: In member function 'virtual void xla::{anonymous}::RefcountingHashMapTest_ForEachEmpty_Test::TestBody()':
tensorflow/compiler/xla/refcounting_hash_map_test.cc:80:3: error: 'int64' was not declared in this scope
   int64 count = 0;
...
```